server/src/main/scala/za/co/absa/atum/server/model/PaginatedResult.scala

```diff
@@ -16,13 +16,27 @@
 
 package za.co.absa.atum.server.model
 
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.{Decoder, Encoder}
+
 sealed trait PaginatedResult[R] {
   def data: Seq[R]
 }
 
 object PaginatedResult {
 
   case class ResultHasMore[R](data: Seq[R]) extends PaginatedResult[R]
+
+  object ResultHasMore {
```

Comment:
Remove these objects. We do not need encoders/decoders for PaginatedResults, we need it only for PaginatedResponse.